dashboard/app/views/admin_reports/retention.html.haml

```diff
@@ -7,6 +7,13 @@
 
 %br
 
+%form{method: 'get'}
+  Script IDs:
+  %input{name: "script_ids", id: "script_ids", value: @scripts.join(", ")}
```

Comment:
nit: would suggest changing these variable names to script_ids and stage_ids (since I would expect scripts to be an array of Script objects)